app/src/main/java/org/listenbrainz/android/presentation/features/yim/YimViewModel.kt

```diff
@@ -262,7 +234,9 @@ class YimViewModel @Inject constructor(private val repository: YimRepository, @A
                 )
                 
             }catch (e: Exception){
-                Toast.makeText(context, "Something went wrong.", Toast.LENGTH_SHORT).show()
+                withContext(Dispatchers.Main){
```

Comment:
We shouldn't show a toast from the viewmodel. A toast is a ui update. These should strictly be added only in the activities/composables.

Reply:
Agreed, took a shortcut here (works) 
:P

Reply:
Apparently now I remember that Dispatchers.Main works the same as runOnUiThread{}. Looks valid to me. If I were to wait for the function to return, it'll freeze the UI or the Main thread.

Reply:
You can usually use a LaunchedEffect to perform UI updates once the function completes in a different thread. But I'll let this slip in this PR merge.